site/public/js/pdf/PDFAnnotateEmbedded.js

```diff
@@ -7,13 +7,7 @@ var documentId = '';
 var PAGE_HEIGHT;
 var NUM_PAGES = 0;
 
-window.RENDER_OPTIONS = {
-    documentId,
-    userId: "",
-    pdfDocument: null,
-    scale: parseFloat(localStorage.getItem('scale')) || 1,
-    rotate: parseInt(localStorage.getItem('rotate')) || 0
-};
```

Comment:
Let's leave this as a global object, and add a new key/value pair `studentPopup: false`.